secrethub/test/secrethub/internal_grpc_api_test.exs

```diff
@@ -835,6 +835,7 @@ defmodule Secrethub.InternalGrpcApi.Test do
       assert Map.get(jwt.fields, "aud") == "https://testera.localhost"
       assert Map.get(jwt.fields, "iss") == "https://testera.localhost"
       assert Map.get(jwt.fields, "sub") == "project:front:pipeline:semaphore.yml"
```

Comment:
It's a nitpick, but that will make the tests better. Can we update the `sub` to align with the documentation and how we are invoking this with Zebra? It will also correspond with what we have in `sub2`

https://github.com/semaphoreio/semaphore/blob/0cc176d567a01946f62959aea06d0855c4662607/zebra/lib/zebra/workers/job_request_factory/open_id_connect.ex#L29

https://docs.semaphore.io/reference/openid
<img width="678" height="406" alt="Image" src="https://github.com/user-attachments/assets/a9db9823-6724-49e8-9f69-1b5a798a40bb" />